src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdClientSettings.Edit.cshtml

```diff
@@ -13,209 +14,210 @@
     <label asp-for="DisplayName" class="form-label">@T["Display Name"]</label>
     <input asp-for="DisplayName" class="form-control" />
     <span asp-validation-for="DisplayName"></span>
-    <span class="hint">@T["Display name of the IdP"]</span>
+    <span class="hint">@T["User-Friendly Display Name of the Identity Provider"]</span>
 </div>
 
 <div class="mb-3" asp-validation-class-for="Authority">
     <label asp-for="Authority" class="form-label">@T["Authority"]</label>
     <input asp-for="Authority" class="form-control" />
     <span asp-validation-for="Authority"></span>
-    <span class="hint">@T["Authority to use when making OpenIdConnect calls"]</span>
+    <span class="hint">@T["Authority URL for OpenID Connect Requests"]</span>
 </div>
 
 <div class="mb-3" asp-validation-class-for="ClientId">
     <label asp-for="ClientId" class="form-label">@T["ClientId"]</label>
     <input asp-for="ClientId" class="form-control" />
     <span asp-validation-for="ClientId"></span>
-    <span class="hint">@T["the 'client_id'"]</span>
+    <span class="hint">@T["The '{0}' assigned by the identity provider.", "client_id"]</span>
 </div>
 
 <div class="mb-3" asp-validation-class-for="CallbackPath">
     <label asp-for="CallbackPath" class="form-label">@T["CallbackPath"]</label>
     <input asp-for="CallbackPath" placeholder="/signin-oidc" class="form-control" />
     <span asp-validation-for="CallbackPath"></span>
-    <span class="hint">@T["The request path within the application's base path where the user-agent will be returned. The middleware will process this request when it arrives."]</span>
+    <span class="hint">@T["The request path, relative to the application's base path, where the user-agent will be redirected. The middleware will handle this request upon arrival."]</span>
 </div>
 
 <div class="mb-3" asp-validation-class-for="SignedOutCallbackPath">
     <label asp-for="SignedOutCallbackPath" class="form-label">@T["SignedOut CallbackPath"]</label>
     <input asp-for="SignedOutCallbackPath" placeholder="/signout-callback-oidc" class="form-control" />
     <span asp-validation-for="SignedOutCallbackPath"></span>
-    <span class="hint">@T["The request path within the application's base path where the user agent will be returned after sign out from the identity provider."]</span>
-    <a class="hint" href="http://openid.net/specs/openid-connect-session-1_0.html#RedirectionAfterLogout" target="_blank">@T["See post_logout_redirect_uri from "]http://openid.net/specs/openid-connect-session-1_0.html#RedirectionAfterLogout</a>
+    <span class="hint">@T["The request path within the application's base path where the user-agent will be redirected after signing out from the identity provider."]</span>
+    <span class="hint">
+        @T["To see 'post_logout_redirect_uri' from, visit: <a href=\"{0}\" target=\"_blank\">", "http://openid.net/specs/openid-connect-session-1_0.html#RedirectionAfterLogout"]</a>
```

Comment:
Sounds grammatically weird, no?

Reply:
when I read it again yes. Because I read `form` instead of `from` :). I'll fix it.

Reply:
fixed